src/Business Foundation/App/NoSeries/src/Single/NoSeries.Codeunit.al

```diff
@@ -326,6 +326,20 @@ codeunit 310 "No. Series"
         exit(NoSeriesImpl.GetNoSeriesLine(NoSeriesLine, NoSeriesCode, UsageDate, HideErrorsAndWarnings));
     end;
 
+    /// <summary>
+    /// Filters the No Series Lines and selects the currently applicable line based on the workdate.
+    /// </summary>
+    /// <param name="NoSeries">The No. Series for which to select a line.</param>
+    /// <param name="NoSeriesLine">The No Series Line is returned filtered and the currently applicable line is selected</param>
+    /// <param name="ResetFilters">If set to true, the filters on the Starting Date and Open fields are reset. The currently applicable line is still selected in the table.</param>
+    /// <returns>Returns true if a line was selected, false if no line was selected.</returns>
+    procedure SelectCurrentNoSeriesLine(NoSeries: Record "No. Series"; var NoSeriesLine: Record "No. Series Line"; ResetFilters: Boolean): Boolean
```

Comment:
Maybe call ResetFilters for ResetStartingDateAndOpenFilters :P. More seriously though, it's a bit confusing that it's simply called reset filters even though it only resets filters for two fields. Maybe just always put this to true and specify that is what this function does. Still seems odd to me. I get that we need these reset for later use, but this function shouldn't care about what it should be used for but what the name implies it should do, which isn't to reset filters just in case of other functions using it..
Maybe the other functions should reset these fields instead? That might make sense.. not sure, I haven't looked at that code.
Anyway this is not a big problem and feel free to won't fix this and check in.

Reply:
Yeah... unless someone proposes a nice name or solution, i'll leave it as is for now...